src/Valet.UnitTests/AppTests.cs

```diff
@@ -35,7 +35,7 @@ public void AfterEachTest()
     }
 
     [TestCase("4256ea72fd01deac3e967f6b19f907587dcd6f0a976301f1aecc73dc6f146a4a", "4256ea72fd01deac3e967f6b19f907587dcd6f0a976301f1aecc73dc6f146a4a", "")]
-    [TestCase("4256ea72fd01deac3e967f6b19f907587dcd6f0a976301f1aecc73dc6f146a4a", "67eed1493c461efd993be9777598a456562f4e0c6b0bddcb19d819220a06dd4b", "A new version of the Valet CLI is available. Run 'gh valet update' to update.\n")]
+    [TestCase("4256ea72fd01deac3e967f6b19f907587dcd6f0a976301f1aecc73dc6f146a4a", "67eed1493c461efd993be9777598a456562f4e0c6b0bddcb19d819220a06dd4b", $"A new version of the Valet CLI is available. Run 'gh valet update' to update.")]
```

Comment:
Is the `$` needed here? (I would suggest the change but it isn't letting me for some reason

Reply:
Nothing is interpolated within the string, so I would think no.